### ISSUE TITLE
Incorrect answer on NIA formula

### ISSUE THREAD

**Issue by muchang** [CVC4/CVC4#3413]
Hi,
For this formula:
```
(declare-fun a () Int)
(declare-fun e () Int)
(declare-fun f () Bool)
(assert (= (div a e) (- 1)))
(assert (= f (not (= e (- 1)))))
(assert (ite f false (= (div a (- 1)) (- 1))))
(check-sat)
``` 
CVC4 reports `unsat` while Z3 reports `sat`

OS: Ubuntu 18.04
Revision: cvc4 1.8-prerelease [git HEAD 3af2dfea] assertions:on

**Comment by 4tXJ7f** [comment]
It looks like this issue is fixed by #3415 as well.

**Comment by 4tXJ7f** [comment]
This issue triggers the same bug as #3412. I've added both versions as regressions.

**Comment by ajreynol** [comment]
Closed due to #3415.
